<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugConsole</class>
 <widget class="QDialog" name="DebugConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug Console</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="selectLocal">
       <property name="text">
        <string>local</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="selectCore">
       <property name="text">
        <string>core</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="scriptEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="evalButton">
     <property name="text">
      <string>Evaluate!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>selectLocal</tabstop>
  <tabstop>selectCore</tabstop>
  <tabstop>scriptEdit</tabstop>
  <tabstop>evalButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
